Source: Written by Calvin Walton for Exherbo
Upstream: TODO
Reason: Configure script is missing check for "ar" tool, resulting in it using
  the unprefixed "ar" on Exherbo, when a prefixed tool is required.

--- radvd-2.19.old/configure.ac 2022-09-07 11:35:09.602969136 -0400
+++ radvd-2.19/configure.ac     2022-09-07 11:35:31.392991022 -0400
@@ -49,6 +49,7 @@
 dnl Determine CC and preset CFLAGS
 AC_PROG_CC_C99
 AC_PROG_RANLIB
+AM_PROG_AR
 
 AC_ARG_WITH([stack-protector], AS_HELP_STRING([--without-stack-protector], [Build without -fstack-protector]),[],[with_stack_protector=yes])
 AM_CONDITIONAL(HAVE_STACK_PROTECTOR, test x"$with_stack_protector" = xyes)

